fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/InclusionQuerySegmentAggregator.java

```diff
@@ -412,4 +423,23 @@ private void addBindVariables(List<Object> bindVariables) {
 
         log.exiting(CLASSNAME, METHODNAME);
     }
+
+    /**
+     * Adds the appropriate limit clause to the passed query string buffer,
+     * based on the type of database we're running against.
+     *
+     * @param queryString A query string buffer.
+     */
+    private void addLimitClause(StringBuilder queryString) {
+        int limit = Integer.MAX_VALUE;
+        if (this.pageSize <= Integer.MAX_VALUE - (SearchConstants.MAX_PAGE_SIZE + 1)) {
+            limit = this.pageSize + SearchConstants.MAX_PAGE_SIZE + 1;
+        }
+
+        if (this.parameterDao.isDb2Database()) {
+            queryString.append(" LIMIT ").append(limit);
```

Comment:
do we have a Constant for this ? I thought there was one? maybe we add one for 440,442's references

Reply:
Constants added for LIMIT, OFFSET, FETCH FIRST, FETCH NEXT, ROWS, ROWS ONLY